### PR TITLE
[RFR] Handle JS errors

### PR DESCRIPTION
**Pull request by manuquentin**

### CODE REVIEW

File: app/View/Show.js [hunk 1]
```diff
@@ -52,8 +52,6 @@ class ShowView extends React.Component {
     }
 
     render() {
-        if (this.state.data.get('pending')) return null;
```
Comment (jeromemacias) [inline app/View/Show.js]: `pending` is not necessary anymore ?

Comment (manuquentin) [inline app/View/Show.js]: Nope, I've removed every `pending` state

File: app/View/List.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+'use strict';
```
Comment (jeromemacias) [inline app/View/List.js]: Is it needed ?

Comment (manuquentin) [inline app/View/List.js]: Nope

**Pull request merged**